pkg/report/output/output.go

```diff
@@ -36,7 +36,15 @@ func GetOutput(
 	case flag.ReportDataFlow:
 		output, err = GetDataflow(report, config, false)
 	case flag.ReportSecurity:
-		output, err = reportSecurity(report, config, baseBranchFindings)
+		var securityOutput *types.Output[security.OutputData]
+		securityOutput, err = reportSecurity(report, config, baseBranchFindings)
+
+		output = &types.Output[security.Results]{
+			Data:         securityOutput.Data.Findings,
+			Dataflow:     securityOutput.Dataflow,
+			Files:        securityOutput.Files,
+			ReportFailed: securityOutput.ReportFailed,
+		}
```

Comment:
`reportSecurity` returns data with `Findings` and `IgnoredFindings` (which are used by SaaS report). To exclude ignored findings from the rest of the CLI and the various report formats (JSON, YAML), we manipulate the output of `reportSecurity` so that output data is simply `security.Results` and does not include ignored findings.